Source/SwiftLintFramework/Models/MasterRuleList.swift

```diff
@@ -100,17 +33,17 @@ public let masterRuleList = RuleList(rules:
     FileHeaderRule.self,
     FileLengthRule.self,
     FirstWhereRule.self,
+    ForWhereRule.self,
```

Comment:
Sourcery returns these already sorted, and I don't think there's a way to re-sort. Is this a problem?